server/liveblog/blogs/tasks.py

```diff
@@ -21,11 +21,12 @@
 logger = logging.getLogger('superdesk')
 
 
-def publish_embed(blog_id, api_host=None, theme=None):
+def publish_embed(blog_id, theme=None, output=None, api_host=None):
```

Comment:
this is handling the `multiple themes` cases a swell which is directly  linked to the publish url on the s3.